codespell_lib/data/dictionary.txt

```diff
@@ -17547,6 +17564,7 @@ poeple->people
 poer->power
 poety->poetry
 pogress->progress
+poindcloud->pointcloud
```

Comment:
Seems a bit specific?

Reply:
Why? E.g. 3D lidar scanner for automated driving would deliver a point cloud (but 2 words)

Reply:
Yeah that's what I meant, I wouldn't object to two words, apart from Codespell can't currently do them, and poind->point, which isn't currently in there, so can we add it please @luzpaz .

Reply:
https://github.com/search?q=pointcloud&type=Code  
https://github.com/search?q=poindcloud&type=Code  
I think it merits being added. I will also add `poind->point`

Reply:
Yeah okay you've convinced me, thanks for adding poind too.